contracts/GenerativeArtworks.sol

```diff
@@ -160,4 +163,32 @@ contract GenerativeArtworks is ERC721Enumerable {
     function pieceShowAllPrints(uint pieceId) external view returns (uint256[] memory) {
         return pieceIdToPrintIds[pieceId];
     }
+
+    function updateAdditionalPayee(uint256 pieceId, address additionalPayeeAddress, uint256 additionalPayeePercentage) external onlyAdmin {
+        require(additionalPayeePercentage <= 100, "Percentage must be <= 100"); // TODO ensure total percentages <= 100
+        uint i;
+        bool found;
+        uint totalPercentage = 0;
+        for (i = 0; i < pieceIdToAdditionalPayees[pieceId].length; i++) {
+            if (pieceIdToAdditionalPayees[pieceId][i] == additionalPayeeAddress) {
```

Comment:
pieceIdToAdditionalPayees[pieceId][i] can you make this into a variable it may make this if statement easier to read?